src/test/java/nextstep/subway/line/domain/SectionsTest.java

```diff
@@ -0,0 +1,186 @@
+
+    private static Station 양평역;
+    private static Station 영등포구청역;
+    private static Station 영등포시장역;
+    private static Station 신길역;
+    private static Station 오목교역;
+
+    private static Line 오호선;
+    private static Supplier<Section> 구간_영등포구청역_신길역 = () ->  new Section(1L, 오호선, 영등포구청역, 신길역, 기본_구간_거리_30);
+    private static Supplier<Section> 구간_영등포구청역_영등포시장역 = () -> new Section(2L, 오호선, 영등포구청역, 영등포시장역, 절반_구간_거리_15);
+    private static Supplier<Section> 구간_양평역_신길역 = () -> new Section(3L, 오호선, 양평역, 신길역, 기본_구간_거리_30);
+    private static Supplier<Section> 구간_오목교역_영등포구청역 = () -> new Section(4L, 오호선, 오목교역, 영등포구청역, 기본_구간_거리_30);
+    private static Supplier<Section> 구간_양평역_영등포구청역 = () -> new Section(5L, 오호선, 양평역, 영등포구청역, 기본_구간_거리_30);
+
+    private static Sections sections = new Sections();
+
+    @BeforeEach
+    void 구간_생성() {
+        양평역 = new Station(1L, "양평역");
+        영등포구청역 = new Station(2L, "영등포구청역");
+        영등포시장역 = new Station(3L, "영등포시장역");
+        신길역 = new Station(4L, "신길역");
+        오목교역 = new Station(5L, "오목교역");
+
+        오호선 = new Line(1L, "5호선", "보라색", 영등포구청역, 신길역, 기본_구간_거리_30);
+        sections = new Sections();
+    }
+
+    @Test
+    void add_성공_케이스1() {
+        // given
+        Stations stations = new Stations(asList(영등포구청역, 신길역));
+
+        // when
+        sections.add(구간_영등포구청역_신길역.get());
+
+        // then
+        assertThat(sections.size()).isEqualTo(1);
+        assertThat(sections.toStations()).isEqualTo(stations);
+    }
+
+    @Test
+    void add_성공_케이스2() {
+        // given
+        Stations stations = new Stations(asList(양평역, 영등포구청역, 영등포시장역, 신길역));
+
+        // when
+        sections.add(구간_영등포구청역_신길역.get());
+        sections.add(구간_영등포구청역_영등포시장역.get());
+        sections.add(구간_양평역_영등포구청역.get());
+
+        // then
+        assertThat(sections.size()).isEqualTo(3);
+        assertThat(sections.toStations()).isEqualTo(stations);
+    }
+
+    @Test
+    void add_예외_중복된_구간() {
+        // given
+        Section duplicatedSection = 구간_양평역_영등포구청역.get();
+
+        // when
+        sections.add(duplicatedSection);
+        sections.add(구간_영등포구청역_신길역.get());
+
+        // Then
+        assertThatExceptionOfType(UnaddableSectionException.class)
+                .isThrownBy(() -> sections.add(duplicatedSection));
+    }
+
+    @Test
+    void add_예외_거리부족() {
+        // given
+        Section shortDistanceSection = 구간_양평역_영등포구청역.get();
+
+        // when
+        sections.add(구간_오목교역_영등포구청역.get());
+
+        // Then
+        assertThatExceptionOfType(BelowZeroDistanceException.class)
+                .isThrownBy(() -> sections.add(shortDistanceSection));
+    }
+
+    @Test
+    void toStations_성공() {
+        // given
+        Stations stations = new Stations();
+
+
+        // when
+        sections.add(구간_영등포구청역_신길역.get());
+        sections.add(구간_영등포구청역_영등포시장역.get());
+        sections.add(구간_양평역_영등포구청역.get());
+
+        // then
+        assertThat(sections.size()).isEqualTo(3);
```

Comment:
가장 중요한 단위 테스트 👍 
아래처럼 검증해보면 더 좋겠네요 :)

```suggestion
        // then
        assertThat(sections.size()).isEqualTo(3);
        assertThat(sections.toStations().get()).containsExactly(양평역, 영등포구청역, 영등포시장역, 신길역);
```

Reply:
말씀 해주신 내용 참고해서 개선해보겠습니다!